src/sql-lexer/src/keywords.rs

```diff
@@ -23,6 +23,13 @@ use std::str::FromStr;
 
 use uncased::UncasedStr;
 
+#[cfg(target_arch = "wasm32")]
+use wasm_bindgen::prelude::*;
+
+#[cfg(target_arch = "wasm32")]
+#[wasm_bindgen(typescript_custom_section)]
+const KEYWORDS_TS_DEF: &'static str = r#"export function getKeywords(): string[];"#;
```

Comment:
Unfortunately there isn't a nice way to do this. This is explicitly declared because WASM bindgen infers `get_keywords` as returning `any[]` or `Array` (if using js-sys).